src/gloss/data/primitives.clj

```diff
@@ -148,6 +198,13 @@
    :uint16-le (primitive-codec .getShort .putShort 2 short->ushort short ushort->short :le)
    :uint16-be (primitive-codec .getShort .putShort 2 short->ushort short ushort->short :be)
 
+   :uint24 (primitive-codec (partial get-unsigned-medium (native-byte-order))
+                            (partial put-medium (native-byte-order)) 3 identity int identity)
+   :uint24-le (primitive-codec (partial get-unsigned-medium :le)
+                               (partial put-medium :le) 3 identity int identity :le)
```

Comment:
Not sure if the last `:le` here still needed because I'm dealing with the byteorder manually

Reply:
Yeah, it might not be. For the built-in supported types, the final `bo` parameter is used to force the buf to a certain order, write out the value in the current order, and then flip the order back, to force writing LE values in a BE buffer (or vice versa).

But you're manually writing out the bytes in sequential order, and `medium->bytes/bytes->umedium` take care of the order before writing, flipping the order should have no effect. I think you can leave the `:le` and `:be` out, but add an explanatory comment above, saying _why_ they're not necessary in this case.